sass/px-dropdown.scss

```diff
@@ -223,6 +223,9 @@ $inuit-enable-btn--primary: true;
 .item-icon {
   --iron-icon-width: var(--px-dropdown-item-icon-size, calculateRem(16px));
   --iron-icon-height: var(--px-dropdown-item-icon-size, calculateRem(16px));
+  
+  -webkit-margin-end: $inuit-base-spacing-unit--small;
+  -moz-margin-end: $inuit-base-spacing-unit--small;
```

Comment:
this change won't affect ie/edge - if all we're looking for is to add padding to the right of icons, we should be using `margin-right: $inuit-base-spacing-unit--small;` - that is unless the RTL changes are affecting this in a negative way.

if this change is intended to "reset" the RTL updates from `4.9.1`, would it be smarter to make this change in another way?

Reply:
`4.9.1`only added `margin-left` for the dropdown **button**'s icon, it doesn't handle the dropdown **item** icon.
Since in `4.9.1` we removed `px-spacing-design` import, the item icon lost `margin-right` value, it caused below regression.
![image](https://user-images.githubusercontent.com/1411188/53213931-a814da80-3685-11e9-9a7f-62a48e7770aa.png)

And why my fix not use `margin-right`  instead of `margin-end`, it because the RTL changes will affect it, we need to handle RTL case too.
**LTR**
![image](https://user-images.githubusercontent.com/1411188/53214054-19548d80-3686-11e9-8c94-9dfc81441667.png)

**RTL**
![image](https://user-images.githubusercontent.com/1411188/53214063-1eb1d800-3686-11e9-8860-a9b934c73914.png)

Reply:
From a cross-browser perspective - I feel that the `-webkit-margin-start/end` method of fixing this issue is not the correct way to fix this issue; I fully realize that you are only building on the changes that we started in the previous release.

the UI team is working on a different way of fixing the issue in a cross browser aligned method.